### ISSUE TITLE
"s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left." output again and again. 

### ISSUE THREAD

**Issue by wertercatt** [bibanon/tubeup#37]
Log:
```
powerkitten@powerkitten-GA-78LMT-S2P:~$ tubeup "http://jp.channel.pandora.tv/channel/video.ptv?c1=&ch_userid=jpchan15&prgid=44933722"
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2017.05.09
[debug] Python version 3.5.2 - Linux-4.4.0-57-generic-x86_64-with-elementary_OS-0.3.2-freya
[debug] exe versions: ffmpeg 2.4.3-1ubuntu1, ffprobe 2.4.3-1ubuntu1
[debug] Proxy map: {}
[debug] Public IP address: 2605:a601:a20:9500:d1a0:e13c:1342:4e7d
{'_elapsed_str': '13:52', 'elapsed': 832.6274771690369, 'total_bytes': 416286757, 'downloaded_bytes': 416286757, '_total_bytes_str': '397.00MiB', 'status': 'finished', 'filename': '/home/powerkitten/.tubeup/downloads/Super_Mario_64_120_star_speed_run_1_46_35_by_nero-44933722.flv'}
:: Downloaded: /home/powerkitten/.tubeup/downloads/Super_Mario_64_120_star_speed_run_1_46_35_by_nero-44933722.flv...
:: Uploading /home/powerkitten/.tubeup/downloads/Super_Mario_64_120_star_speed_run_1_46_35_by_nero-44933722...
2017-06-04 17:16:57,503 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:17:28,316 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:17:59,033 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:18:29,795 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:19:00,613 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:19:31,396 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:20:02,121 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:20:32,892 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:21:03,670 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:21:34,421 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:22:05,183 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:22:36,030 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:23:06,804 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:23:37,591 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:24:08,396 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:24:39,168 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:25:09,934 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:25:40,715 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:26:11,412 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:26:42,112 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:27:12,897 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:27:43,656 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:28:14,453 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:28:45,241 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:29:15,983 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:29:47,243 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:30:18,014 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:30:49,176 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:31:19,946 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:31:50,700 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:32:21,442 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:32:52,179 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:33:22,950 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:33:53,712 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:34:24,469 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:34:55,224 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:35:25,986 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:35:56,713 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:36:27,525 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:36:58,869 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:37:29,624 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:38:00,393 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:38:31,146 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:39:01,916 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:39:32,703 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:40:03,462 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:40:34,224 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:41:05,100 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:41:35,831 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:42:06,584 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:42:37,324 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:43:08,411 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:43:39,185 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:44:09,976 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:44:40,760 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:45:11,524 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:45:42,263 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:46:13,003 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:46:43,760 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:47:14,613 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:47:45,509 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:48:16,233 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:48:47,000 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:49:17,757 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:49:48,538 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:50:19,308 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:50:50,183 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:51:20,940 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
2017-06-04 17:51:51,666 - internetarchive.item - INFO - s3 is overloaded, sleeping for 30 seconds and retrying. 9001 retries left.
```

**Comment by vxbinaca** [comment]
Not a Tubeup bug, this is actually a safety feature.

https://twitter.com/internetarchive/status/870363499691507712
https://twitter.com/internetarchive/status/870514798064058370
https://twitter.com/internetarchive/status/870521701024452608

I set the internetarchive options to wait a REALLY long time before failing, because Archive.orgs s3-esque services gets either overloaded or goes down sometimes. Add Internet Archive on Twittter and check that before making an issue.

TL;DR: Wait a while. if it's not failing to prompt - and I put a lot of thought into tweaking opts to where that happens as little as possible - then it's still ripping.

Closing.
